docs/content/concepts/tokenomics/gas-in-sui.mdx

```diff
@@ -1,88 +1,88 @@
+75 bytes = 7,500 units
 
-The gas budget for a Sui transaction is defined in SUI units and transactions are successfully executed if:
+Sui’s storage fund model charges storage fees upfront but rebates 99% if the data is later deleted.
 
-`gas_budget >= max{computation_fees,total_gas_fees}`
+Gas Budgets {#gas-budgets}
+Every transaction includes a gas budget—the maximum gas you’re willing to pay.
 
-If the gas budget does not fulfill this condition, then the transaction fails and a portion of the gas budget is charged. In cases where the `gas_budget` is insufficient for covering `computation_fees`, then the entirety of the `gas_budget` is charged. In cases where `gas_budget` is sufficient for covering `computation_fees` but not the `total_gas_fees`, then a portion of the `gas_budget` corresponding to `computation_fees` and the fees associated with mutating the transaction's input objects are charged.
+A transaction is accepted only if:
 
-Ultimately, a successful transaction requires the end user to pay the transaction's `total_gas_fees`. However, since it is challenging to perfectly forecast computation time before the transaction is processed, the `gas_budget` condition also requires the `gas_budget` to be at least as large as the transaction's `computation_fees` in case the transaction aborts. In some cases -- especially in the presence of high storage rebates, and, thus negative net storage fees -- the gas budget might be higher than the total gas fees you pay.
+gas_budget >= max(computation_fees, total_gas_fees)
 
-Importantly, the minimum gas budget is 2,000 MIST (.000002 SUI). This ensures validators are compensated with at least 2,000 MIST even if the gas budget is incorrectly specified and the transaction aborts. Additionally, this protects the Sui network from being spammed with a large number of transactions with minimal gas budgets. The maximum gas budget is 50 billion MIST or 50 SUI. This protects the network against overflow of internal multiplications and gas limits for denial of service attacks.
+What if it’s too low?
+If the gas budget < computation_fees → the entire budget is charged, and the transaction fails.
 
-As mentioned previously, the storage rebate currently equals 99% of the originally paid storage fees. Because the gas budget applies to the totality of gas fees, it is often the case that a transaction only goes through if the gas budget is considerably higher than the net gas fees that a user ultimately pays.
+If gas budget is enough for computation but not storage → computation fees and mutation costs are charged.
+
+The minimum gas budget is 2,000 MIST (0.000002 SUI)
+The maximum is 50 billion MIST (50 SUI)
+
+Description | Gas Price | Compute CU | Storage Price | Storage Units | Rebate | Min Budget | Net Fees
+Simple tx, 10 bytes storage | 1,000 | 1,000 | 75 | 1,000 | 0 | 1.075M | 1.075M
```

Comment:
```suggestion
| Basic transaction, 10 bytes storage | 1,000 MIST | 1,000 | 75 MIST | 1,000 | 0 MIST | 1,075,000 MIST | 1,075,000 MIST |
```
Don't use tx as it's not certain that people know what it stands for. Don't use mix of digits and abbreviations. Prefer the long form of numbers to remove any doubt. Include units of measure (or intro the table with 'all price values are in MIST' or similar).